docs/detections/detections-ui-exceptions.asciidoc

```diff
@@ -152,25 +152,22 @@ Closes all alerts that match the exception's conditions and were generated only
 
 [float]
 [[endpoint-rule-exceptions]]
-=== Add Elastic Endpoint Security exceptions
+=== Add Elastic {endpoint-sec} exceptions
 
-Like detection rule exceptions, you can add Endpoint agent exceptions via both
-the Elastic Endpoint Security rule and its generated alerts. Alerts generated
-from the Elastic Endpoint Security rule have the following fields:
+Like detection rule exceptions, you can add Endpoint agent exceptions either by editing the Elastic {endpoint-sec} rule or as an action on a generated {endpoint-sec} alert. Alerts generated from the Elastic {endpoint-sec} rule have the following fields:
 
 * `kibana.alert.original_event.module determined:endpoint`
 * `kibana.alert.original_event.kind:alert`
 
-Additionally, you can add Endpoint exceptions via rules that are associated
-with Elastic endpoint rule exceptions. To associate rules, when creating or
-editing a rule select the
-<<rule-ui-advanced-params, _Elastic endpoint exceptions_>> option.
+You can also add Endpoint exceptions to rules that are associated with Elastic Endpoint rule exceptions. To associate rules, when creating or editing a rule, select the <<rule-ui-advanced-params, Elastic Endpoint exceptions>> option.
 
 [IMPORTANT]
 =============
-Exceptions added to the Elastic Endpoint Security rule affect all alerts sent
+Exceptions added to the Elastic {endpoint-sec} rule affect all alerts sent
 from the Endpoint agent. Be careful not to unintentionally prevent some Endpoint
 alerts.
+
+Additionally, to add an Endpoint exception to the Elastic {endpoint-sec} rule, there must be at least one {endpoint-sec} alert generated in the system. For non-production use, you can trigger a test alert using malware emulation techniques or tools, such as the Anti Malware Testfile from the https://www.eicar.org/[European Institute for Computer Anti-Virus Research (EICAR)].
```

Comment:
Any thoughts on the EICAR testfile recommendation? It's mentioned in the originating [bug PR](https://github.com/elastic/kibana/issues/121475) & [issue](https://github.com/elastic/security-docs/issues/1360), but IMO it seems like a workaround for an edge case bug that we're mentioning in docs mostly just in case someone encounters it. But on the other side, it might still be a helpful testing strategy to recommend?